packages/remix-routes/src/__tests__/cli.test.ts

```diff
@@ -16,3 +16,10 @@ test('build v2 routes', async () => {
     fs.readFileSync(path.resolve(__dirname, '../../fixture/v2/node_modules/remix-routes.d.ts'), 'utf8'),
   ).toMatchSnapshot();
 });
+
+test("build customPaths routes", async () => {
+  await build(path.resolve(__dirname, "../../fixture/customPaths"), { outputDirPath: "./types/remix-routes", watch: false, strict: false });
+  expect(
+    fs.readFileSync(path.resolve( __dirname, "../../fixture/customPaths/types/remix-routes/remix-routes.d.ts"), "utf8"),
```

Comment:
You can try printing out the content of remix-routes.d.ts to see what's generated on windows.

Reply:
Next run should show that now.

I'm hoping it's not something like line-feed differences between linux & Windows.

Reply:
Argh. Sorry for the copy-paste error on the v1 route console log. Pushed the correct file path to log.